django/template/base.py

```diff
@@ -151,6 +151,9 @@ def __eq__(self, other):
             self.loader == other.loader
         )
 
+    def __ne__(self, other):
```

Comment:
FWIW I made sure we don't forget to remove this method [when we drop support for Python 2](https://code.djangoproject.com/ticket/23919).